### PR TITLE
DLT-FT Integration

### PR DESCRIPTION
**Pull request by kruss**

### CODE REVIEW

**Review by DmitryAstafyev — CHANGES_REQUESTED** [review]

File: application/apps/rustcore/ts-bindings/spec/defaults.json [hunk 1]
```diff
@@ -173,6 +173,22 @@
                     }
                 }
             }
+        },
+        "dlt": {
+            "regular": {
+                "execute_only": [1, 2, 3, 4, 5, 6, 7, 8, 9],
```
[the remark this answers is not in the document]
Comment (kruss) [reply application/apps/rustcore/ts-bindings/spec/defaults.json]: done

File: application/apps/rustcore/ts-bindings/src/native/native.dlt.ts [hunk 2]
```diff
@@ -21,3 +23,120 @@ export function stats(files: string[]): Promise<Types.StatisticInfo> {
...
+                uuid = undefined;
+            });
+        self.canceled(() => {
+            if (uuid === undefined) {
+                return;
+            }
+            const _uuid = uuid;
+            uuid = undefined;
+            dlt.cancelOperation(_uuid).catch((err: unknown) => {
+                reject(new Error(error(err)));
```
[the remark this answers is not in the document]
Comment (kruss) [reply application/apps/rustcore/ts-bindings/src/native/native.dlt.ts]: done

File: application/apps/rustcore/ts-bindings/src/native/native.dlt.ts [hunk 3]
```diff
@@ -21,3 +23,120 @@ export function stats(files: string[]): Promise<Types.StatisticInfo> {
...
+            })
+            .catch((err: unknown) => {
+                reject(new Error(error(err)));
+            })
+            .finally(() => {
+                uuid = undefined;
+            });
+        self.canceled(() => {
+            if (uuid === undefined) {
+                return;
```
[the remark this answers is not in the document]
Comment (kruss) [reply application/apps/rustcore/ts-bindings/src/native/native.dlt.ts:56]: done

File: application/apps/rustcore/ts-bindings/src/native/native.dlt.ts [hunk 4]
```diff
@@ -21,3 +23,120 @@ export function stats(files: string[]): Promise<Types.StatisticInfo> {
...
+            }
+            const _uuid = uuid;
+            uuid = undefined;
+            dlt.cancelOperation(_uuid).catch((err: unknown) => {
+                reject(new Error(error(err)));
```
[the remark this answers is not in the document]
Comment (kruss) [reply application/apps/rustcore/ts-bindings/src/native/native.dlt.ts]: done

File: application/apps/rustcore/ts-bindings/src/native/native.dlt.ts [hunk 5]
```diff
@@ -21,3 +23,120 @@ export function stats(files: string[]): Promise<Types.StatisticInfo> {
...
+            })
+            .catch((err: unknown) => {
+                reject(new Error(error(err)));
+            })
+            .finally(() => {
+                uuid = undefined;
+            });
+        self.canceled(() => {
+            if (uuid === undefined) {
+                return;
```
[the remark this answers is not in the document]
Comment (kruss) [reply application/apps/rustcore/ts-bindings/src/native/native.dlt.ts:98]: done

**Pull request closed without merging**